### ISSUE TITLE
Differences in MINOR and PATCH parts are not reported

### ISSUE THREAD

**Issue by rimutaka** [serayuzgur/crates#141]
A new version alert is not displayed if the difference is in MINOR or PATCH numbers only. It seems that only MAJOR version differences result in displaying the latest version number.

E.g., `urlencoding` has the following versions available:
``` 
1.3.3(docs)
1.2.0
1.1.1
1.1.0
1.0.0
0.2.0
0.1.1
```
Using `urlencoding = "1.0.0"` through to `urlencoding = "1.3.3"` prints a green tick and no latest version alert.
![image](https://user-images.githubusercontent.com/5926028/118895658-aae10180-b95a-11eb-98c1-4b4bc9cd4a46.png)

Dropping to `urlencoding = "0.2"` displays `X 1.3.3` next to it.
![image](https://user-images.githubusercontent.com/5926028/118895592-8e44c980-b95a-11eb-9270-c383dad96c04.png)

Is this a bug or how it should work?

I would expect anything below `urlencoding = "1.3.3"` to have the latest version alert printed next to it.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20
 - Version v0.5.9

**Comment by phoekz** [comment]
I agree. At least it could be an option whether or not minor/patch differences trigger the latest version alert.

**Comment by koutheir** [comment]
This looks like a regression, because I remember this used to work correctly, even when minor versions change.

**Comment by serayuzgur** [comment]
> think it's a breaking change and we'll need a configuration setting for it to tun on/off?
> Would you consider a PR or prefer to do it yourself? I have no experience with VSCode extension, but happy to give it a tr

I will check and implement it if it is simple to

**Comment by Christoph-AK** [comment]
Hi, any progress on this? 

Maybe add a less aggressive warning on this, maybe ⚠️ instead of ❌?
